index.js

```diff
@@ -52,7 +52,7 @@ module.exports = {
         eventHandlerPropPrefix: 'on',
       },
     ],
-    'react/jsx-indent': ['error', 2],
+    'react/jsx-indent': 'off',
     'react/jsx-indent-props': ['error', 2],
```

Comment:
should this also be disabled?

Reply:
True

Reply:
Done

Reply:
are you aware of `eslint-config-prettier-check`?

Reply:
It doesn't detect these rule, @lbebber . It happens because these rules are not printed in command `eslint --print-config`. But I don't really know why they are not printed.

Reply:
doesn't this work? `eslint --print-config .eslintrc | eslint-config-prettier-check`

Reply:
Actually, in this case, the command is `eslint --print-config index.js | eslint-config-prettier-check`, but no, doesn't work for the reason I mentioned above. The rules must be printed, but these are not.

Reply:
@estacioneto huh, `npx eslint --print-config index.js` does print the config for me. 🤷‍♂

Reply:
ah, nevermind, I misunderstood you. yes it does work but these specific rules are not printed. weird

Reply:
found the issue. lets discuss over DM